Dockerfile

```
@@ -8,10 +8,10 @@ RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -trimpath -o /usr/b
 # Runtime stage
 FROM alpine:3.21 as release
 
-RUN adduser -D -u 12345 -g 12345 k6
+RUN adduser -D -u 12345 -g 12345
```

I think this line is fine as it was, the user needs a name regardless of whether we refer to it by ID or by name.

```suggestion
RUN adduser -D -u 12345 -g 12345 k6
```